### PR TITLE
CCM-6416 front-end tests for error transformation

### PR DESCRIPTION
**Pull request by adadigital**
## Summary

This adds frontend tests for the CCM-6416 bugfix which ensures that the correct error response is returned for requests where the personalisation field is too large

The changes to the targets are temporary and will be reverted before merging

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Checklist
* [x] Brief description of work completed, and any technical decisions made as part of the PR
* [x] PR link added as a comment to the relevant JIRA ticket
* [x] PR link shared on Slack and/or Teams
* [x] 2 reviews received
* [x] Tester approval

### CODE REVIEW

File: proxies/live/apiproxy/targets/target.xml [hunk 1]
```diff
@@ -18,10 +18,7 @@
         <SSLInfo>
             <Enabled>true</Enabled>
         </SSLInfo>
-        <LoadBalancer>
-            <Server name="{{ TARGET_SERVER_OVERRIDE | default('communications-manager-target') }}"/>
-        </LoadBalancer>
-        <Path>{requestpath}</Path>
+        <URL>https://comms-apim.de-adsm6.communications.national.nhs.uk</URL>
```
Comment (simonlabarere) [inline proxies/live/apiproxy/targets/target.xml:21]: Please can you revert these changes :)

**Review by simonlabarere — APPROVED** [review]

File: proxies/live/apiproxy/targets/target.xml (hunk 1 above)
Comment (adadigital) [reply proxies/live/apiproxy/targets/target.xml:21]: Yes I will revert them when I have approval, I just didn't want to revert them only to need to un-revert them to test any changes that anyone requested.

**Review by DavidFueryBJSS — APPROVED** [review]

**Review by danh-nhs — APPROVED** [review]
